main.py

```diff
@@ -22,7 +24,17 @@
 game_ids = list(yesterdays_games_data['GAME_ID'])
 
 # Retrieve play by play data from those games, sorted by game chronologically
-playbyplay = PlayByPlay(game_ids)
+raw_plays_dfs = []
+for game_id in game_ids:
+    pbp = playbyplayv2.PlayByPlayV2('00' + str(game_id))
+    raw_plays_dfs.append(pbp.get_data_frames()[0])
+    time.sleep(0.5)
+
+cursor.execute("SELECT MAX(play_id) FROM playbyplay")
```

Comment:
This would be a pretty big change, but instead of using raw SQL queries, you should consider using libraries like this https://www.sqlalchemy.org/ http://www.sqlobject.org/. 

This way it works more like code than strings. check them out and see what they do.

Reply:
I just discovered SQLalchemy the other day and it does seem like a much easier way of interacting with a DB. I will work on changing from psycopg2.